### ISSUE TITLE
Unable to access/interact with combo boxes in certain web sites.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2842]
**Reported by Harich on 2012-11-30 23:10**
Few web sites I've visited seems to give trouble when using NVDA. These web sites have combo boxes and edit fields that I am unable to access/interact with using NVDA. To test my issue, please visit http://amerigroup.prismisp.com/ and you'll find what I meant. The said web site is for searching doctors in United States. First, NVDA would not read the content of the combo boxes even though the focus mode is on. Second, you would lose focus as soon as you tab to a next combo/edit field. Finally, there is a third combo box (if starting from the top of the page) entitled "Select a Product", you cannot even open it despite having focus mode turned on. Please note that I'm using Internet Explorer ten.

**Comment by ehollig** [comment]
It seems to me, filling out this form using IE 11 and Windows 10 seems to work as expected.
The values I used were, "Nevada" and "Las Vegas, NV". I was able to change the value of the "Product" combo box and all other combo boxes on the page. If this is still the case on Windows 8, this can be reopened, but for now Closing.
